styles/global.css

```diff
@@ -3,7 +3,7 @@
   --secondary-text-color: rgba(0, 0, 0, 0.7);
   --link-color: #0080ff;
   --link-decoration-color: rgba(0, 128, 255, 0.3);
-  --font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI',
```

Comment:
@nburka `BlinkMacSystemFont` is a keyword value rather than a proper font name, so it's not supposed to be in quotes. I think this is a bug in the stylelint config. That said, it works in quotes in Chrome, which is all that matters, so: good fix 👍